### PR TITLE
Cloud Users page revisions

### PR DESCRIPTION
**Pull request by jaffrepaul**
Copy, url & image updates

### CODE REVIEW

File: docs/guides/cloud/users.mdx [hunk 1]
```diff
@@ -7,22 +7,24 @@ A user is anyone who logs in to Cypress Cloud.
 
 ## Invite users
 
-You can invite users from [Cypress Cloud](https://on.cypress.io/cloud). Invited
-users will see all projects and tests run for the organization.
+Organization [owner or admin](#User-roles) roles can invite users from
+[Cypress Cloud](https://on.cypress.io/cloud). Those invited will see all
```
Comment (conversayShawn) [inline docs/guides/cloud/users.mdx:11]: Organization [owner or admin](#User-roles) roles can invite [Cypress Cloud](https://on.cypress.io/cloud) users. Those invited will see all

**Review by elylucas — APPROVED** [review]

File: docs/guides/cloud/users.mdx [hunk 2]
```diff
@@ -112,14 +113,11 @@ deny them access.
 
 ## User updates
 
-Changes can be made to both the primary email address associated with your
-Cypress Cloud account and the email address for billing notifications.
+Updates to email addresses requires an **Owner** or **Admin** role.
```
[the remark this answers is not in the document]
Comment (jaffrepaul) [reply docs/guides/cloud/users.mdx:116]: 👍

File: docs/guides/cloud/users.mdx (hunk 1 above)
Comment (jaffrepaul) [reply docs/guides/cloud/users.mdx:11]: 👍

**Review by conversayShawn — APPROVED** [review]
All looks good!

**Pull request merged**